powershell/definitions/Q.d.ts

```diff
@@ -0,0 +1,386 @@
+// Type definitions for Q
```

Comment:
I just copied these definitions from the vso-agent-tasks repo. It seemed to be required to make "gulp test" and the psm.RunPS() work. I don't really know what I'm doing here, so just wanted to double-check this is correct thing to do.